src/middlewared/middlewared/plugins/smb.py

```diff
@@ -715,6 +750,7 @@ async def do_create(self, data):
             'datastore.insert', self._config.datastore, data,
             {'prefix': self._config.datastore_prefix})
 
+        await self.strip_comments(data)
```

Comment:
We would now be returning data with comments stripped?

Should we change return to use get_instance and remove .extend call?

Reply:
Yeah, that's a good idea.